app/tests/integration/conftest.py

```diff
@@ -162,18 +162,12 @@ def openstack_connection_fixture(cloud_name: str) -> Connection:
     return openstack.connect(cloud_name)
 
 
-@pytest.fixture(scope="module", name="dockerhub_mirror_url")
-def dockerhub_mirror_url_fixture(pytestconfig: pytest.Config) -> str | None:
+@pytest.fixture(scope="module", name="dockerhub_mirror")
+def dockerhub_mirror_fixture(pytestconfig: pytest.Config) -> urllib.parse.ParseResult | None:
```

Comment:
this is still there because we have a test which runs a docker command in the image, which may get rate-limited